docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+If you lost the data itself, then no, there’s no way to recover that, and you will be slashed for it. If the data itself is recoverable, though (say you just missed a Windowed PoSt), then the Recovery process will let you regain the sector.
+
+### Has Filecoin confirmed the use of SDR algorithm? Is there any evidence of malicious construction?
+
+SDR is confirmed for mainnet launch, and we have no evidence of malicious construction. The algorithm is also going through both internal and external security audits.
+
+If you have any information about any potential security problem or malicious construction, reach out to our team at security@filecoin.org.
+
+### How likely is the NSE algorithm to come on line later? Is AMD's configuration the most consistent with the current SDR algorithm?
+
+About NSE: NSE is one of the best candidates for a proof upgrade, and teams are working on implementation. But there are other candidates too, which are promising as well. It may be that another algorithm ends up better than NSE -- we don’t know yet. Proof upgrades will arrive after mainnet launch and will coexist.
+
+Yes, AMD may be optimal for SDR, [see this for why](https://github.com/filecoin-project/lotus/blob/master/documentation/en/sealing-procs.md)
+
+### How are you working to bootstrap the demand side of the marketplace? The Discover program is nice but who is the target market for users and how do you get them?
+
+In addition to Filecoin Discover, a number of groups are actively building tools and services to support adoption of the Filecoin network with developers and clients. For example, check out the recordings from our [Virtual Community Meetup](https://filecoin.io/blog/filecoin-virtual-community-meetup-recap/) to see updates about Textile Powergate and Starling Storage.
+
+A number of other programs, like Filecoin Ignite, will also contribute additional demand, tooling, and usage.
+
+### Why not implement the scheme that the white paper says "miners and clients automatically match orders through order books"？Is it because the LIMITS of TPS?
+
+There will be off-chain orderbooks and miner marketplaces -- some are in development now from some teams. They will work mostly off-chain, because TPS on chain is not enough for the volume of usage we expect on Filecoin. These orderbooks build on the basic deal-flow on chain. These orderbooks will arrive in their own development trajectory -- most likely around or soon after mainnet launch.
+
+### Why does Filecoin mining work best on AMD?
+
+Currently, Filecoin's Proof of Replication (PoRep) prefers to be run on AMD processors. More accurately, it runs much much slower on Intel CPUs (it runs competitively fast on some ARM processors, like the ones in newer Samsung phones, but they lack the RAM to seal the larger sector sizes). The main reason that we see this benefit on AMD processors is due to their implementation of the SHA hardware instructions. Now, why do we use the SHA instruction?
```

Comment:
Currently, Filecoin's Proof of Replication (PoRep) prefers to be run on AMD processors. See [this description of Filecoin sealing](https://github.com/filecoin-project/lotus/blob/master/documentation/en/sealing-procs.md) for more information. More accurately, it runs much much slower on Intel CPUs. It runs competitively fast on some ARM processors, like the ones in newer Samsung phones, but they lack the RAM to seal the larger sector sizes. The main reason that we see this benefit on AMD processors is due to their implementation of the SHA hardware instructions.